.github/actions/security-controls/action.yml

```diff
@@ -0,0 +1,23 @@
+name: "Generate github issues"
+description: "Create github issues for each controls provided in the csv file"
+
+inputs:
+  token:
+    description: "Token for allowing the action to create issues"
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout Code
+      uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
```

Comment:
I have a feeling this won't work as expected - my gut feeling is that it'll just attempt to checkout the repo that this action has been added to.  

Can you do a quick test from another repo to see what happens?

Reply:
Actually, this should fix it - you can specify the `repository` that you want to checkout on the action:
https://github.com/actions/checkout#usage

Reply:
Can confirm this will just checkout the repo the action is a part of, if you are looking at checking out a seperate repo take a peak at the docs specifically this section: https://github.com/actions/checkout#checkout-multiple-repos-side-by-side

Reply:
Ah nice, that `path` attribute will be perfect.

Reply:
Thanks!

Reply:
Done! Thanks for the help!